### PR TITLE
docs(ImageCroppingWidget): Use clipping planes

### PR DESCRIPTION
**Pull request by floryst**
### Changes

Update ImageCroppingWidget to use the volume clip planes.

### Results

Cropping still works, but we no longer use the ImageCropFilter for this example.

### Testing

Nothing here for now.

### CODE REVIEW

File: Sources/Widgets/Widgets3D/ImageCroppingWidget/example/index.js [hunk 1]
```diff
@@ -125,25 +127,53 @@ actor.getProperty().setDiffuse(0.7);
...
-    window.asdf = widget;
     const cropState = widget.getWidgetState().getCroppingPlanes();
     cropState.onModified(() => {
-      cropFilter.setCroppingPlanes(cropState.getPlanes());
+      const planes = getCroppingPlanes(image, cropState.getPlanes());
+      mapper.removeAllClippingPlanes();
+      planes.forEach((plane) => {
+        mapper.addClippingPlane(plane);
+      });
+      mapper.modified();
```
Comment (floryst) [inline Sources/Widgets/Widgets3D/ImageCroppingWidget/example/index.js:176]: `addClippingPlane` does not call `publicAPI.modified()`. @jourdain would it be worth it to update `addClippingPlane` to call modified? The only perf case I can think of: not calling `modified` on every add means we don't trigger a render when adding multiple planes.

Comment (jourdain) [reply Sources/Widgets/Widgets3D/ImageCroppingWidget/example/index.js:176]: We could have a debounced modified call too?

Comment (jourdain) [reply Sources/Widgets/Widgets3D/ImageCroppingWidget/example/index.js:176]: maybe with an optional argument (batch=true)

**Comment by thewtex** [comment]
nice!

File: Sources/Widgets/Widgets3D/ImageCroppingWidget/example/index.js (hunk 1 above)
Comment (floryst) [reply Sources/Widgets/Widgets3D/ImageCroppingWidget/example/index.js:176]: For now I will not bother with making changes to `addClippingPlane`. However, I will make an issue to reference this potential change.

**Pull request merged**